census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -361,7 +363,7 @@ public void inboundMessageRead(
   }
 
   @VisibleForTesting
-  final class ServerTracerFactory extends ServerStreamTracer.Factory {
+  public final class ServerTracerFactory extends ServerStreamTracer.Factory {
```

Comment:
Why are these two classes now public?

Reply:
[`AbstractServerImplBuilderTest`](https://github.com/grpc/grpc-java/blob/6517ac8a617c9d0ee6d1f87f4c36aafca049b5a4/core/src/test/java/io/grpc/internal/AbstractServerImplBuilderTest.java#L55) is doing some test verifying the type of tracer factory applied to the server. Similar for client side.

Reply:
Meh. We can compare the class names as strings.